src/urllib3/contrib/_securetransport/bindings.py

```diff
@@ -45,28 +44,44 @@
     c_bool,
 )
 from ctypes import CDLL, POINTER, CFUNCTYPE
+from ctypes.util import find_library
 
-
-security_path = find_library("Security")
-if not security_path:
-    raise ImportError("The library Security could not be found")
-
-
-core_foundation_path = find_library("CoreFoundation")
-if not core_foundation_path:
-    raise ImportError("The library CoreFoundation could not be found")
+from urllib3.packages.six import raise_from
 
 
 version = platform.mac_ver()[0]
 version_info = tuple(map(int, version.split(".")))
-if version_info < (10, 8):
+if platform.system() != "Darwin" or version_info < (10, 8):
```

Comment:
This checks need to be done before looking at `version_info`, because right now computing `version_info` raises an exception on Linux and Windows.

Reply:
Fixed.

Was this failing before? There was _no_ check for whether it was macOS before.

Reply:
Importerror was raised if find_library() didn't return.

Reply:
Right. Can you please raise ImportError instead of OSError? This is the expected exception indeed